Source/WebInspectorUI/UserInterface/Views/DOMNodeDetailsSidebarPanel.js

```diff
@@ -546,6 +547,19 @@ WI.DOMNodeDetailsSidebarPanel = class DOMNodeDetailsSidebarPanel extends WI.DOMD
                         checked = WI.UIString("No");
                 }
 
+                let switchState = "";
+                // COMPATIBILITY (macOS X.0, iOS X.0): DOM.AccessibilityProperties.switchState did not exist yet.
+                if (InspectorBackend.Enum.DOM.AccessibilityPropertiesSwitchState) {
+                    switch (accessibilityProperties.switchState) {
+                    case InspectorBackend.Enum.DOM.AccessibilityPropertiesSwitchState.On:
+                        switchState = WI.UIString("Switch State @ On");
```

Comment:
unfortunately this is not how `WI.UIString` works.  it's `WI.UIString(string, key, comment)`, so in this case you'd probably want something like
```js
WI.UIString("On", "On @ Switch State", "Label indicating that a input of type switch is on.");
```

Reply:
Ahh, thank you for catching this!